source/projects/webRadio/de/index.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: project
+description: Raspberry Pi, Automation, Hacking & Web Radio
+lang: de
+title: Automation & Hacking mit dem Raspberry Pi
+subtitle: Das neueste Tutorial des Ideation & Protyping Lab beleuchtet die Tiefen der Automatisierung und erklärt "Wie man sein Bluetooth-Radio zum Autoplay Web Radio hackt."
+type: publication
+colorMode: bright
+thumbnail: /projects/webRadio/images/thumbnail_second.jpg
+heroImage: /projects/webRadio/images/hero.jpg
+socialMediaImage: /projects/webRadio/images/featuredImage.jpg
+visible: true
+featured: false
+authors:
+  - julia-zimmermann
+start: 2019-11-12
+end: 2019-11-13
+status: ongoing
+date: 2019-11-13
+
+assets:
+  css:
+    - ../styles/index.css
+
+materialsIncluded:
+  - name: "GitHub"
+    link: "https://github.com/technologiestiftung/werkstatt/blob/master/HowTo_WebRadio.md"
+
+redirect_from:
+  - /projects/webRadio/index.html
+
+---
+
+*Auch das Ideation & Prototyping Lab möchten im Zeitalter der zunehmenden Automation Schritt halten und hat daher das hauseigene Bluetooth-Radio des [CityLAB Berlin](https://www.citylab-berlin.org/) zum Web Radio transformatiert. Das dabei entstandene Tutorial beschäftigt sich mit dem **Raspberry Pi**, dessen **Autostart-Konfiguration** & **Bluetooth Radios**. Wie das Alles zusammenhängt und wie man sich mit Hilfe eines RasPi's ein Autoplay Web Radio hackt, das soll in diesem Artikel beschrieben werden. Das ausführlich Tutorial findet ihr auf [GitHub](https://github.com/technologiestiftung/werkstatt/blob/master/HowTo_WebRadio.md)*.
+
+## Die Idee
+In Zeiten der zunehmenden Automation wollten wir einmal mehr hinter die Kulissen blicken und ein bluetooth-fähiges Radio mit Hilfe eines Raspberry Pi 3 und einem **Autostart-Programm** zu einem Autoplay Web Radio hacken. Die Kommunikation vom Laptop zum RasPi erfolgt dabei über SSH - vom RasPi zum Radio über Bluetooth. Die gesamte Konfiguration (Bluetooth, VLC-Mediaplayer Command, Delays etc.) wurde in ein **Shell-Skript** geschrieben, welches schließlich mit der Autostart-Datei des Raspberry's verknüpft wurde. 
```

Comment:
I would suggest to add at the fist  occurrence of `Raspberry Pi` the abbreviation `(RasPi)`

Reply:
added abbreviation within the teaser